.github/workflows/skip-validation.yml

```diff
@@ -0,0 +1,45 @@
+name: Validation
+
+on:
+  push:
+    branches: [ main, '2.1', '2.0', '1.3' ]
+    paths:
+      - 'hilla-logo.svg'
+      - 'README.md'
+      - '.github/ISSUE_TEMPLATE/*'
+      - 'packages/**/README.md'
+  pull_request_target:
+    paths:
+      - 'hilla-logo.svg'
+      - 'README.md'
+      - '.github/ISSUE_TEMPLATE/*'
+      - 'packages/**/README.md'
+
+jobs:
+  check:
+    name: Check
```

Comment:
By the way, shouldn't it be a separate workflow? I mean, it is about user permission checking, I believe, it has nothing in common with the testing workflow. Isn't it so?

Reply:
The intention here is to prevent running any further steps in the testing workflow until the checks are satisfied. A separate workflow would bring more complexity around triggering one workflow from the other. I don't think it's worth it, compared to just few extra steps in the first job in the same file.

Now that you brought this up, however, I realised that the skip workflow does not run any testing steps anyway. We don't need to check any permissions here then. I removed this unnecessary job.